include/miopen/miopen.h

```diff
@@ -698,7 +711,19 @@ MIOPEN_EXPORT miopenStatus_t miopenSetTensorDescriptor(miopenTensorDescriptor_t
                                                        const int* dimsA,
                                                        const int* stridesA);
 
-/*! @brief Get size of N-dimensional tensor
+/*! @brief Set the tensor cast type
+ *
+ *  For tensors where the cast_type attribute is set, the tensor elements would be converted to the
+ * target type before the target operation is applied. Currently, only supported for convolution
+ * operations targeting the FP8 datatype
```

Comment:
[Q] Are you sure that this is supported only for FP8 output? What about BF8?

[Q] Am I right that this is supported only for FP16 input tensors? Maybe this is worth mentioning?

[Q] Is it so that conversion (by truncation, i.e. without rounding) from FP16 (E5M10) to BF8 (E5M2) is free? Because the implementation can simply skip the low byte of FP16 elements.